test/fixtures/layer-import-atrules.expected.css

```diff
@@ -0,0 +1,127 @@
+@media screen and (min-width: 320px) {
+@layer imported-with-media.level-2.level-3 {
+@layer Z {
+  body {
+    color: cyan;
+  }
+}
+}
+}
+
+@media screen {
+@layer imported-with-media.level-2 {
+
+@layer Y {
+  body {
+    color: purple;
+  }
+}
+}
+}
+
+@media screen {
```

Comment:
Why do we close the previous `@media screen` just to reopen a new one right afterwards?

Reply:
This is the same pattern as we see here : https://github.com/postcss/postcss-import/blob/master/test/fixtures/media-content.expected.css#L3

The `at rules` seem to be closed after each file import.
This is existing behaviour.

Reply:
Haha, I'd forgotten how we handled media queries. This is fine then.